x-pack/plugins/lens/public/indexpattern_datasource/layerpanel.tsx

```diff
@@ -26,13 +26,19 @@ export function LayerPanel({ state, layerId, onChangeIndexPattern }: IndexPatter
   const notFoundTitleLabel = i18n.translate('xpack.lens.layerPanel.missingDataView', {
     defaultMessage: 'Data view not found',
   });
-
+  const triggerLabel = state.indexPatternRefs.reduce(
```

Comment:
`indexPattern` should have the readable title as well - just map it in here: https://github.com/elastic/kibana/blob/d364f237c5dd2309eb7e61c3c46399048719fa25/x-pack/plugins/lens/public/indexpattern_datasource/loader.ts#L90